CHANGELOG.md

```diff
@@ -15,6 +16,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 - `ContractedUnitPrice` column
 - `RegionId` column
 - `RegionName` column
+- `ConsumedQuantity` column
+- `ConsumedUnit` column
```

Comment:
We should capture this as either an add/remove or a change. Currently, I captured this as a 'change' since we mostly treated it as a rename + update. It's in line 41-48 below.

Reply:
It's probably easier to track historical changes if we treat every rename as a remove/add. Otherwise, we'll lose track in the summary table. I would request to keep the list sorted alphabetically, tho.